extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -381,6 +381,7 @@ public List<? extends com.fasterxml.jackson.databind.Module> getJacksonModules()
   @AfterEach
   public void tearDown2()
   {
+    Mockito.framework().clearInlineMocks();
```

Comment:
I don't think this leak-like usage is msq specific ; it could possibly be added to `CalciteTestBase` so that all `Calcite*Test` excercise this call